common/src/Model/Brush.cpp

```diff
@@ -344,25 +351,25 @@ namespace TrenchBroom {
             }
         }
 
-        BrushFace* Brush::findFace(const Vec3& normal) const {
+        BrushFace* Brush::findFace(const vm::vec3& normal) const {
             for (auto* face : m_faces) {
-                if (face->boundary().normal.equals(normal)) {
+                if (isEqual(face->boundary().normal, normal, vm::constants<FloatType>::almostZero())) {
                     return face;
                 }
             }
             return nullptr;
         }
 
-        BrushFace* Brush::findFace(const Plane3& boundary) const {
+        BrushFace* Brush::findFace(const vm::plane3& boundary) const {
             for (auto* face : m_faces) {
-                if (face->boundary().equals(boundary)) {
+                if (isEqual(face->boundary(), boundary, vm::constants<FloatType>::almostZero())) {
```

Comment:
same as above, I'd suggest `isPlaneAlmostEqual`. the normal part of the plane is in [-1..+1] and the 'distance' component of the plane might be in +-256k so it might be desirable to use different epsilons (and help ensure the same approximate plane comparison function is used everywhere)

Reply:
But we already have an isEqual function for planes. We can change it to take into account the different epsilons by taking an extra parameter, but for consistency reasons I'd like to have the same name for all the isEqual functions (later to be renamed to isAlmostEqual).